cttportfolio.css

```diff
@@ -3,168 +3,481 @@
 }
 html{
 	font-size:62.5%;
+	font-family:'Montserrat', sans-serif;
+	color:rgba(4,8,9,1);
 }
 body{
 	margin:0;
+	background-color:rgb(2,8,9);
 }
 main > div{
 	margin:0 auto;
-	max-width:1350px;
-	padding:0 1%;
+	max-width:1300px;
+}
+main{
+}
+h4, h5, h6,.pp_body h1{
+	font-weight:700;
+	font-size:6.0rem;
+	text-transform:uppercase;
+    -webkit-text-stroke-width: 0.3px;
+    -webkit-text-stroke-color: black;
```

Comment:
Indentation is messed up here.

Reply:
it wasnt on mine,  ithink maybe cos i copied it, it was tabs vs spaces idk